components/homepage/Logo.tsx

```diff
@@ -1,12 +1,11 @@
 import Image from 'next/image'
 import { satoshiBold } from '@/util/localFonts';
 
-
 const Logo = () => {
     return (
         <div className={satoshiBold.className + " mt-28 md:mt-56 lg:mt-48 pr-3 z-40"}>
-            <Image src='/icons/CAI.webp' width={500} height={500} className="object-contain z-0 align-bottom select-none hidden md:block" alt="CAI Logo for desktop"/>
-            <Image src='/icons/CAI.webp' width={290} height={290} className="object-contain z-0 align-bottom select-none md:hidden block" alt="CAI Logo for mobile"/>
+            <Image src='/Icons/CAI.webp' width={500} height={500} className="object-contain z-0 align-bottom select-none hidden md:block" alt="CAI Logo for desktop"/>
```

Comment:
just curious, why the capitalization changes for `icons`

Reply:
At some point the folder name got switched to `Icons` from `icons` but now it doesn't make much sense with every other folder being lowercase. I'll swap that over to be consistent.